pyvista/core/cell.py

```diff
@@ -356,13 +356,34 @@ def bounds(self) -> Tuple[float, float, float, float, float, float]:
 
         Examples
         --------
-        >>> import pyvista
-        >>> mesh = pyvista.Sphere()
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
         >>> mesh.get_cell(0).bounds
         (-0.05405950918793678, -5.551115123125783e-17, 0.0, 0.011239604093134403, -0.5, -0.49706897139549255)
+
         """
         return self.GetBounds()
 
+    @property
+    def center(self) -> Tuple[float, float, float]:
+        """Get the center of the cell.
+
+        Examples
+        --------
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
+        >>> mesh.get_cell(0).center
+        (-0.0356, 0.00375, -0.498)
+
+        """
+        para_center = [0.0, 0.0, 0.0]
+        sub_id = self.GetParametricCenter(para_center)
+        sub_id = _vtk.mutable(0)
```

Comment:
If the variable `sub_id` in the first line is not used, the expression can be simple.
```suggestion
        _ = self.GetParametricCenter(para_center)
        sub_id = _vtk.mutable(0)
```

Reply:
Good catch! I think that we need to pass it into the following line.  I'm not sure why it needs to be mutable however, which is why I originally set it to 0.

Reply:
https://github.com/pyvista/pyvista/pull/4480/commits/0ca25ab5d9deea52b0a57627f33a9945901435f9 should address this